dataframe/html.go

```diff
@@ -0,0 +1,134 @@
+package dataframe
```

Comment:
I would rather have this on the dataframe.go file

Reply:
Updated.